src/main/java/graphql/schema/GraphQLDirective.java

```diff
@@ -51,8 +52,9 @@ public GraphQLDirective(String name,
                             List<GraphQLArgument> arguments,
                             boolean onOperation,
                             boolean onFragment,
-                            boolean onField) {
-        this(name, description, locations, arguments, onOperation, onFragment, onField, null);
+                            boolean onField,
+                            boolean onVariableDefinition) {
+        this(name, description, locations, arguments, onOperation, onFragment, onField,onVariableDefinition,null);
```

Comment:
we should not add new constructors.

Reply:
that is not a new constructor, but added new field(onVariableDefinition) in constructor